services/monitoring/grafana/terraform/.terraform-version

```diff
@@ -0,0 +1 @@
+1.5.1
```

Comment:
is this the right place for the .terraform-version file? Should we rather have only one tf version for the whole repo, at the repo base dir, and maybe even have a pre-commit hook that enforces only one single tf version? not sure